src/main/java/io/levelops/plugins/levelops_job_reporter/plugins/LevelOpsPluginImpl.java

```diff
@@ -30,7 +30,7 @@ public class LevelOpsPluginImpl {
     private static final String DATA_DIR_NAME = "run-complete-data";
     public static final String PLUGIN_SHORT_NAME = "propelo-job-reporter";
     private String levelOpsApiKey = "";
-    private String levelOpsPluginPath = "${JENKINS_HOME}/levelops-jenkin";
+    private String levelOpsPluginPath = "${JENKINS_HOME}/levelops-jenkins";
```

Comment:
If we change this, it would affect the existing plugins that upgrade. The path can stay as-is. Customers do not see it or navigate to it.

Reply:
done